File: Prereq-OOP/src/Human.java

// Classes contain instructions for how objects can be created as well as how objects carry out certain actions
public class Human {

    String name, eyeColor;
    int age, heightInInches;

    public void speak() {
        System.out.println("Hello, my name is " + name);
        System.out.println("I am " + heightInInches + " inches tall");
        System.out.println("I am" + age + " years old");
        System.out.println("My eye color is " + eyeColor);
    }

    public void eat() {
        System.out.println("eating...");
    }

    public void walk() {
        System.out.println("walking...");
    }

    public void work() {
        System.out.println("working...");
    }
}

File: Python/Earth/human.py
class Human:

    # Constructor
    def __init__(self, name, age, heightInInches, eyeColor):
        self.name = name
        self.age = age
        self.heightInInches = heightInInches
        self.eyeColor = eyeColor

    def speak(self):
        print('Hello, my name is', self.name)
        print('I am', self.heightInInches, 'inches tall')
        print('I am', self.age, 'years old')
        print('My eye color is', self.eyeColor)

    def eat(self):
        print('eating...')

    def walk(self):
        print('walking...')

    def work(self):
        print('working...')